recipes/inline-model-metrics/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "inline-model-metrics" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inline_model_metrics-{{ version }}.tar.gz
+  sha256: 0f0ffef1ae2d5450698f926e79124a49de2edc540a62122d6b59754d05fa5fb8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6,<3.10
+    - pip
+  run:
+    - python >=3.6,<3.10
+    - metoffice-afterburner
+    - iris
+    - netcdf4
+    - numpy
+    - tempest-helper
+
+test:
+  imports:
+    - inline_model_metrics
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
You can remove the call to `pip check`. This is unlikely ever to work due to the difference in naming of `iris` vs. `scitools-iris` for conda-forge vs. PyPI. The import check itself seems pretty robust for this package to make sure it's properly working and has all dependencies installed.

```suggestion
```